public/pages/QueryInsights/QueryInsights.js

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Same question here? The standard around plugins are to use typescript for type support